packages/augur-ui/src/modules/auth/components/connect-dropdown/connect-dropdown.tsx

```diff
@@ -176,7 +176,7 @@ const ConnectDropdown = (props: ConnectDropdownProps) => {
       logo: EthIcon,
       value: ethReserveInDai,
       subValue: reserveEthAmount.formattedValue,
-      disabled: GsnEnabled ? (balances.signerBalances?.eth || 0) === 0 : false,
+      disabled: GsnEnabled ? balances.signerBalances.eth !== "0" : false,
```

Comment:
This might cause a TypeError: Cannot read property 'eth' of undefined. I suggest re-adding the "?." / optional chaining.

Reply:
it won't the reducer is defaulted to have a value. the `?` isn't needed.